rpcs3/rpcs3qt/tooltips.h

```diff
@@ -194,6 +194,7 @@ class Tooltips : public QObject
 		const QString enter_button_assignment = tr("The button used for enter/accept/confirm in system dialogs.\nChange this to use the Circle button instead, which is the default configuration on Japanese systems and in many Japanese games.\nIn these cases having the cross button assigned can often lead to confusion.");
 		const QString enable_host_root        = tr("Required for some Homebrew.\nIf unsure, don't use this option.");
 		const QString limit_cache_size        = tr("Automatically removes older files from disk cache on boot if it grows larger than the specified value.\nGames can use the cache folder to temporarily store data outside of system memory. It is not used for long-term storage.");
+		const QString console_time_offset     = tr("Sets the time to be used within the console, this will be applied as an offset that tracks wallclock time.\nCan be reset to current wallclock time by clicking \"Set to Now\".");
```

Comment:
isn't wallclock spelled wall clock ?

```suggestion
		const QString console_time_offset     = tr("Sets the time to be used within the console, this will be applied as an offset that tracks wall clock time.\nCan be reset to current wall clock time by clicking \"Set to Now\".");
```